sdk/communication/Azure.Communication.Identity/tests/SessionRecords/CommunicationIdentityClientLiveTests/GetTokenForTeamsUserWithEmptyTokenShouldThrowAsync.json

```diff
@@ -1,38 +1,35 @@
 {
   "Entries": [
     {
-      "RequestUri": "https://sanitized.communication.azure.net/teamsUser/:exchangeAccessToken?api-version=2021-10-31-preview",
+      "RequestUri": "https://jolovcommunication-communication.communication.azure.com/teamsUser/:exchangeAccessToken?api-version=2021-10-31-preview",
```

Comment:
Uri's were inconsistently sanitized. They shouldn't need to be sanitized anyway.